src/main/java/com/github/tomakehurst/wiremock/matching/MatchesJsonPathPattern.java

```diff
@@ -110,6 +112,8 @@ protected MatchResult isAdvancedMatch(String value) {
     } catch (SubExpressionException e) {
       notifier().info(e.getMessage());
```

Comment:
We could just drop this completely. I'm not sure it's really helping, and I plan to remove all of these at some point anyway now that we've got sub-events.